src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
   trace was running (`Activity.Current != null`).
   ([#4890](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4890))
 
+* Fix `TraceContextPropagator` behavior when trace parent flags contains
```

Comment:
You could rephrase this to something like this:
```md
* Fix `TraceContextPropagator` by validating the first digit of the hex-encoded `trace-flags` field of the `traceparent` header.
```

Reply:
Fixed: d0df3dbb0ffd83b08def71a65dd95c96cc26e0bf